test/correct_output/multi_column_var_dz_with_well.out.press.00010.pfb.dist

```diff
@@ -0,0 +1 @@
+0
```

Comment:
Dist files are not normally committed, are these dist files need for some reason?